contracts/BodhiCandy.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.4.18;
+
+import './libs/SafeMath.sol';
+
+contract BodhiCandy {
+    using SafeMath for uint256;
+
+    // Amount in Satoshi to deposit
+    uint256 public constant depositAmount = 1000000;
+
+    // Amount of blocks it takes for the last person to deposit to win if no others deposit within the length.
+    uint256 public constant winningBlockLength = 30;
```

Comment:
Set it 25. And you can merge and deploy it

Reply:
You got my comments in Slack about the fallback function not working right? I am going to revert the fallback function logic back so it returns the funds when calling it.